llvm/test/CodeGen/DirectX/Metadata/exports.ll

```diff
@@ -0,0 +1,19 @@
+; RUN: opt -S -dxil-metadata-emit %s | FileCheck %s
+
+target triple = "dxilv1.3-unknown-shadermodel6.3-library"
+
+define void @"?f1@@YAXXZ"() #0 {
+entry:
+  ret void
+}
+
+define void @"?f2@MyNamespace@@YAXXZ"() #0 {
+entry:
+  ret void
+}
+
```

Comment:
Is there a function you can add here that isn't exported (lacks `#0`), and thus won't show up in dx.exports below?

Reply:
The test was removed.